src/routes/jira/workspaces/jira-workspaces-get.ts

```diff
@@ -1,3 +1,74 @@
-export const JiraWorkspacesGet = () => {
-	// Content included in ARC-2115-search-connected-orgs
+import { Request, Response } from "express";
+import { Subscription } from "models/subscription";
+import { RepoSyncState } from "models/reposyncstate";
+import { Errors } from "config/errors";
+
+type GitHubWorkspace = {
+	id: string,
+	name: string
+}
+
+const {
+	MISSING_JIRA_HOST,
+	MISSING_SUBSCRIPTION,
+	MISSING_ORG_NAME,
+	NO_MATCHING_WORKSPACES
+} = Errors;
+
+const findMatchingOrgs = async (subscriptions: Subscription[], orgName: string): Promise<GitHubWorkspace[]>  => {
+	const matchingRepos = await RepoSyncState.findByOrgNameAndSubscriptionId(subscriptions, orgName);
+	const matchedOrgs = matchingRepos.map(org => {
+		const { subscriptionId, repoOwner } = org;
+
+		return {
+			id: subscriptionId.toString(),
+			name: repoOwner,
+			// default to false until support is added for createContainer
+			canCreateContainer: false
+		};
+	})
+		.filter((value, index, self) =>
```

Comment:
So if there are multiple repositories with the same `subscriptionId`, it returns only one match. Is this correct? 
Whats the reason behind this? Shouldn't we be returning all the possible matches?

Reply:
+1 a bit confusing in the logic here
Assume the logic is to return a unique org name, we can do a `distinct` query in postgress on the `repoOwner` column right? Postgres will be easier to process it.